homeassistant/components/myuplink/binary_sensor.py

```diff
@@ -25,6 +26,17 @@
     },
 }
 
+CONNECTED_BINARY_SENSOR_DESCRIPTION = BinarySensorEntityDescription(
+    key="connected_state",
+    device_class=BinarySensorDeviceClass.CONNECTIVITY,
+)
+
+ALARM_BINARY_SENSOR_DESCRIPTION = BinarySensorEntityDescription(
+    key="has_alarm",
+    device_class=BinarySensorDeviceClass.PROBLEM,
+    name="Alarm",
```

Comment:
Can we use entity translations?

Reply:
Yes, we can, but we don't use entity translations in this integration because all other entities are named from data in API or by device_class. But it is reasonable to enable translation of this entity name so it behaves similarly to those named by device_class.